Files/Strings/en-US/Resources.resw

```diff
@@ -1818,4 +1818,34 @@
   <data name="SettingsShowMultiselectOption.Header" xml:space="preserve">
     <value>Enable multiselect options (currently only abailable in the tiles and grid view layout modes)</value>
   </data>
+  <data name="DriveTypeCDRom" xml:space="preserve">
+    <value>CD ROM Drive</value>
+  </data>
+  <data name="DriveTypeCloudDrive" xml:space="preserve">
+    <value>Cloud Drive</value>
+  </data>
+  <data name="DriveTypeFixed" xml:space="preserve">
+    <value>Fixed Disk Drive</value>
+  </data>
+  <data name="DriveTypeFloppyDisk" xml:space="preserve">
+    <value>Floppy Disk Drive</value>
+  </data>
+  <data name="DriveTypeNetwork" xml:space="preserve">
+    <value>Network Drive</value>
+  </data>
+  <data name="DriveTypeNoRootDirectory" xml:space="preserve">
+    <value>Unmounted Drive</value>
```

Comment:
Is there a better name for this type?

I found some related discussion at https://stackoverflow.com/q/37321474